src/transport.js

```diff
@@ -14,27 +14,27 @@
  * limitations under the License.
  */
 
-import {Services} from '../../../src/services';
+import {Services} from './services';
```

Comment:
ping @jridgewell @choumx 
What would be a better approach to share code between two extensions.